src/cli/build.ts

```diff
@@ -29,6 +29,10 @@ export default function (program: RootCmd) {
       "-r, --registry-info [<registry>/<username>]",
       "Registry Info: Image registry and username. Note: You must be signed into the registry",
     )
+    .option(
+      "-o, --output-dir [output directory]",
+      "Define where the contents of the dist folder go",
```

Comment:
```suggestion
      "Define where to place build output",
```
...or something like it should read more clearly to someone new (who probably won't know what "contents of the dist folder" means.